### PR TITLE
fix(pfd/fma): merge changes from asobo-1.8.3.0

### PR DESCRIPTION
**Pull request by Benjozork**
See #719

**Summary of Changes**

Merges changes in `Airbus_FMA.js` from `asobo-1.8.3.0` branch.

**Screenshots (if necessary)**

N/A

**Additional context**

N/A

Discord username (if different from GitHub):

### CODE REVIEW

**Comment by Benjozork** [comment]
I left some variable names unclear because I am not sure what Asobo is checking here now.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/Airbus_FMA.js [hunk 1]
```diff
@@ -1446,7 +1449,9 @@ var Airbus_FMA;
             return false;
         }
         GetModeState_APPNAV() {
-            if (Simplane.getAutoPilotAPPRHold() && Simplane.getAutoPilotGlideslopeHold() && Simplane.getAutoPilotApproachType() != 4) {
+            const approachNotLoadedOrSomething = !Simplane.getAutoPilotApproachLoaded() || Simplane.getAutoPilotApproachType() !== 4;
```
Comment (UncleClapton) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/Airbus_FMA.js:1452]: wat...

**Comment by UncleClapton** [comment]
Honestly some of this might be completely obsolete by our own fixes. I would love to get @Curtis-VL to check over this to make sure it's all correct

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/Airbus_FMA.js (hunk 1 above)
Comment (Benjozork) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/Airbus_FMA.js:1452]: See the coment above @UncleClapton

**Comment by Benjozork** [comment]
@UncleClapton Yeah I opened this so that we can have something to test and / or comment on. I do not think any of what they fixed is actually useful, but I opened the PR regardless so that more eyes can go over this.

**Comment by UncleClapton** [comment]
There are also the smaller changes to PFD.css and PFD.js. They should probably be a part of this PR.

**Comment by Benjozork** [comment]
@UncleClapton changes to `A320_Neo_PFD.css` and `A320_Neo_PFD.js` have been added to the PR.

**Comment by JackZ974** [comment]
> 
> 
> There are also the smaller changes to PFD.css and PFD.js. They should probably be a part of this PR.

The only change in PFD.js applies to the horizon which is updated as well. I think this renders the "Fix Horizon" mod obsolete

**Comment by UncleClapton** [comment]
Posting comparison of old and new PFDs

before (pre 1.8 patch, stock plane):


after (Post patch, + mod, + this PR):


Without this PR the positions of each element on the PFD is about the same location. The main thing the CSS changes in this PR is the position of the instruments on the PFD.

**Comment by UncleClapton** [comment]
Tested pretty extensively. Looks like everything is working fine after these changes.

**Comment by JackZ974** [comment]
Why is the Box around MAN TOGA only partial?

**Comment by Benjozork** [comment]
Don't think it was, the only changes are to position.

**Comment by UncleClapton** [comment]
@JackZ974 Box should go all around "MAN TOGA" right?

**Comment by lukecologne** [comment]
I checked out the commit before this PR and the issue is also present there.

**Comment by UncleClapton** [comment]
Come to think of it, yeah this has been around long before the patch.

**Comment by JackZ974** [comment]
> @JackZ974 Box should go all around "MAN TOGA" right?
yes I think so

**Comment by JackZ974** [comment]
> 
> 
> I have that too, not sure if it was introduced by this PR, or if it was there before

guess it was there before

**Comment by UncleClapton** [comment]
TOGA is probably just missing a border style. easy enough fix. Will make (or look for) a ticket on it

**Pull request merged**